Sources/Core/ApiClient/ApiSession.swift

```diff
@@ -29,7 +33,7 @@ extension ApiSession {
           try await withCheckedThrowingContinuation { continuation in
             dataTask = session.dataTask(with: request) { data, response, error in
               if let data, let response {
-                continuation.resume(returning: (data, response))
+                continuation.resume(returning: (data, response as! HTTPURLResponse))
```

Comment:
This was the bit I was most unsure/nervous about re. casting URLResponse to HTTPURLResponse.
I can't throw an exception from here, because this `dataTask` lambda thingy isn't allowed to `throw` AFAIK, so I've used `response as! HTTPURLResponse` which I think means it should panic/crash if that cast is impossible. It should be impossible for it to be impossible, so it should be find, I think :)